packages/block-library/src/cover/edit.js

```diff
@@ -209,44 +209,56 @@ function ResizableCover( {
  *                               color are set, dimRatio is used to decide what is used
  *                               for background darkness checking purposes.
  * @param {?string} overlayColor String containing the overlay color value if one exists.
+ *                               The default is black to match the CSS.
  * @param {?Object} elementRef   If a media background is set, elementRef should contain a reference to a
  *                               dom element that renders that media.
- *
- * @return {boolean} True if the cover background is considered "dark" and false otherwise.
+ * @param {?Function} setAttributes Function to set attributes on the block.
  */
-function useCoverIsDark( url, dimRatio = 50, overlayColor, elementRef ) {
-	const [ isDark, setIsDark ] = useState( false );
+function useCoverIsDark(
+	url,
+	dimRatio = 50,
+	overlayColor = '#000000',
+	elementRef,
+	setAttributes
```

Comment:
I'm not sure if passing in `setAttributes` is the best approach here, but it works :)